app/Achievements/UserFilled100Requests.php

```diff
@@ -15,20 +15,25 @@
 
 use Gstt\Achievements\Achievement;
 
-class UserFilled100Requests extends Achievement
+final class UserFilled100Requests extends Achievement
```

Comment:
I typically do not use `final classes`, but they are considered better practice. 

I personally like leaving the classes a bit more open, because in the real world i think that `final classes` could cause more headaches then what its worth to have them ... 

just my opinion

Reply:
Final classes make sense for things like data classes and models. I'd keep them in, since there's no risk anyone would want to extend them, even in anger. ;)